### PR TITLE
Linux add icon with setWindowIcon

### LINKED ISSUE #4073: No application icon (Linux)
There is no application icon, so the generic gray question mark shows up instead.

### PR DESCRIPTION
**Pull request by dagar**
- fixes #4073

### CODE REVIEW

File: src/main.cc [hunk 1]
```diff
@@ -211,6 +212,8 @@ int main(int argc, char *argv[])
     QGCApplication* app = new QGCApplication(argc, argv, runUnitTests);
     Q_CHECK_PTR(app);
 
+    QApplication::setWindowIcon(QIcon(":/res/resources/icons/qgroundcontrol.ico"));
```
Comment (DonLakeFlyer) [inline src/main.cc]: Shouldn't this be Linux only?

File: qgcresources.qrc [hunk 2]
```diff
@@ -3,11 +3,8 @@
         <file alias="opensans">resources/fonts/OpenSans-Regular.ttf</file>
         <file alias="opensans-demibold">resources/fonts/OpenSans-Semibold.ttf</file>
     </qresource>
-
```
Comment (DonLakeFlyer) [inline qgcresources.qrc]: Your changes are pulling out blank lines and comments.

File: src/main.cc (hunk 1 above)
Comment (dagar) [inline src/main.cc]: Shouldn't it be fine everywhere? The other options for windows and osx set the icon for the binary which is also used for the app if this hasn't been set.

File: qgcresources.qrc (hunk 2 above)
Comment (dagar) [inline qgcresources.qrc]: This is what happens when you use the qtcreator editor.

File: src/main.cc (hunk 1 above)
Comment (DonLakeFlyer) [inline src/main.cc]: Famous last words :) I'd rather not take the chance. Otherwise I'll need to test all OS.

File: qgcresources.qrc (hunk 2 above)
Comment (DonLakeFlyer) [inline qgcresources.qrc]: Hence the reason I never use qtcreator to edit resources files. It's terrible.

File: src/main.cc (hunk 1 above)
Comment (dagar) [inline src/main.cc]: Fair enough.

File: qgcresources.qrc (hunk 2 above)
Comment (dagar) [inline qgcresources.qrc]: Doesn't really matter to me. I've done this several times before, so this time I just kept the change. Your call.

Comment (DonLakeFlyer) [inline qgcresources.qrc]: It guess it's gonna get trashed at some point. The other file is alreaedy trounced anyway.

**Pull request merged**